src/main/java/Duke.java

```diff
@@ -1,10 +1,176 @@
+/**
+ * This is the main class of the chat bot app that helps user remember their different
+ * types of task and "parrot" the task back to them when they request.
+ *
+ * @author YEOWEIHNGWHYELAB
+ */
+
+import java.util.Scanner;
+
 public class Duke {
+    /* Task Counter */
+    public static int numberOfTasks = 0;
+
+    /* This is an array of task & Total task quantity should not exceed 100! */
+    public static Task[] tasks = new Task[100];
+
+    /**
+     * These are text objects created that can print text file
+     * specified into the terminal.
+     */
+    public static PrintTextFile printHelloText = new PrintTextFile("text-art/HelloText.txt");
+    public static PrintTextFile printByeText = new PrintTextFile("text-art/ByeText.txt");
+    public static PrintTextFile printParrotText = new PrintTextFile("text-art/ParrotText.txt");
+    public static PrintTextFile printDukeyText = new PrintTextFile("text-art/DukeyText.txt");
+
+    /**
+     * Prints the current list of task when the "list" command is
+     * entered to the terminal.
+     */
+    public static void printTaskList() {
+        System.out.println("    ____________________________________________________________");
+        for (int i = 0; i < numberOfTasks; i += 1) {
+            System.out.print("     ");
+            System.out.print((i + 1) + ".");
+            tasks[i].printStatus();
+        }
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Adds a new todo task with the description of task provided.
+     *
+     * @param userInputString which contains the "todo" command along with
+     *                        the description of the todo task. No time
+     *                        details is needed here.
+     */
+    public static void addToDo(String userInputString) {
+        String taskName = userInputString.substring(5);
+
+        tasks[numberOfTasks] = new ToDo(taskName);
+        numberOfTasks += 1;
+
+        tasks[numberOfTasks - 1].printAddingStatus(numberOfTasks - 1);
+    }
+
+    /**
+     * Adds a new deadline task with the description of deadline task provided
+     * along with the time which is due "by".
+     *
+     * @param userInputString which contains the "deadline" command along with
+     *                        the description of the deadline task and followed
+     *                        by a "/" to separate the due time "by".
+     */
+    public static void addDeadline(String userInputString) {
```

Comment:
naming of methods makes it easy to understand the code and its purpose. Good job!